dotcom-rendering/src/components/SlideshowCarousel.tsx

```diff
@@ -38,38 +62,141 @@ const caption = css`
-						/>
-						{!!image.imageCaption && (
-							<figcaption css={caption}>
-								{image.imageCaption}
-							</figcaption>
-						)}
-					</figure>
-				</li>
+}) => {
+	const carouselRef = useRef<HTMLOListElement | null>(null);
+	const [previousButtonEnabled, setPreviousButtonEnabled] = useState(false);
+	const [nextButtonEnabled, setNextButtonEnabled] = useState(true);
+
+	const scrollTo = (direction: 'left' | 'right') => {
+		if (!carouselRef.current) return;
+
+		const cardWidth =
+			carouselRef.current.querySelector('li')?.offsetWidth ?? 0;
+		const offset = direction === 'left' ? -cardWidth : cardWidth;
+		carouselRef.current.scrollBy({
+			left: offset,
+			behavior: 'smooth',
+		});
+	};
+
+	/**
+	 * Updates state of navigation buttons based on carousel's scroll position.
+	 *
+	 * This function checks the current scroll position of the carousel and sets
+	 * the styles of the previous and next buttons accordingly. The previous
+	 * button is disabled if the carousel is at the start, and the next button
+	 * is disabled if the carousel is at the end.
+	 */
+	const updateButtonVisibilityOnScroll = () => {
+		const carouselElement = carouselRef.current;
+		if (!carouselElement) return;
+
+		const scrollLeft = carouselElement.scrollLeft;
+		const maxScrollLeft =
+			carouselElement.scrollWidth - carouselElement.clientWidth;
+
+		setPreviousButtonEnabled(scrollLeft > 0);
+		setNextButtonEnabled(scrollLeft < maxScrollLeft);
+	};
+
+	useEffect(() => {
+		const carouselElement = carouselRef.current;
+		if (!carouselElement) return;
+
+		carouselElement.addEventListener(
+			'scroll',
+			updateButtonVisibilityOnScroll,
+		);
+
+		return () => {
+			carouselElement.removeEventListener(
+				'scroll',
+				updateButtonVisibilityOnScroll,
 			);
-		})}
-	</ul>
-);
+		};
+	}, []);
+
+	return (
+		<div>
+			<ul
+				ref={carouselRef}
+				css={carouselStyles}
+				data-heatphan-type="carousel"
+			>
+				{takeFirst(images, 10).map((image, index) => {
+					const loading = index > 0 ? 'lazy' : 'eager';
+					return (
+						<li css={carouselItemStyles} key={image.imageSrc}>
+							<figure>
+								<CardPicture
+									mainImage={image.imageSrc}
+									imageSize={imageSize}
+									aspectRatio="5:4"
+									alt={image.imageCaption}
+									loading={loading}
+								/>
+								{!!image.imageCaption && (
+									<figcaption css={captionStyles}>
+										{image.imageCaption}
+									</figcaption>
+								)}
+							</figure>
+						</li>
+					);
+				})}
+			</ul>
+			<div css={buttonStyles}>
+				<Button
+					hideLabel={true}
+					iconSide="left"
+					icon={<SvgChevronLeftSingle />}
+					onClick={() => scrollTo('left')}
+					priority="tertiary"
+					theme={
+						previousButtonEnabled
+							? themeButton
+							: themeButtonDisabled
+					}
+					size="small"
+					disabled={!previousButtonEnabled}
+					// TODO
+					// aria-label="Previous story"
+					// data-link-name="container left chevron"
```

Comment:
Why not add these now? 🙂

Reply:
Good question 🙂 We left these out in the other carousels as we weren't sure how to label them and there's a separate ticket to revisit and update. That said, there's no real reason to leave them out entirely so I've labelled them in line with the existing carousels on the site and we can amend later as necessary.  I've left the `data-link-name` attributes as _to do_ though as I don't know what these should be at the moment.